### DISCUSSION TITLE
Why I can't update the "Origin allow list" ??

### DISCUSSION THREAD

**Discussion by Flyy-yu** [WalletConnect/walletconnect-monorepo#4116] [Developer Support] [answered]
![Screenshot 2024-01-09 at 6 00 02 PM](https://github.com/WalletConnect/walletconnect-monorepo/assets/25512004/196bd506-d725-4ed0-ad28-5043fa5f7d1d)

**Comment by glitch-txs** [comment]
are you still facing this issue?

**Reply by Flyy-yu** [reply]
Yes. I still can't configure the allowed domains list!

**Reply by glitch-txs** [reply, marked as answer]
so you typed and nothings happens? I think you need to press enter to add domains.

also we have a guide of recomendations here: https://docs.walletconnect.com/cloud/relay#allowlist

**Reply by Flyy-yu** [reply]
Oh, I don't know I was supposed to type there! The drop-down menu greyed out and it gave me the impression that I can't edit it. It works now, thank you.
